interfaces/ros1/wavemap_ros/include/wavemap_ros/inputs/depth_image_input.h

```diff
@@ -46,38 +52,31 @@ struct DepthImageInputHandlerConfig
   //! known (e.g. through calibration) but not corrected by the sensor's driver.
   Seconds<FloatingPoint> time_offset = 0.f;
 
-  //! Name of the topic on which to republish the depth images as pointclouds.
+  //! Name of the topic on which to publish the depth images as pointclouds.
   //! Useful to share the pointclouds with other ROS nodes and for debugging.
   //! Disabled if not set.
-  std::string reprojected_pointcloud_topic_name;
-  //! Name of the topic on which to republish the range image computed from
-  //! the pointclouds. Useful for debugging. Disabled if not set.
-  std::string projected_range_image_topic_name;
+  std::string projected_pointcloud_topic_name;
 
   static MemberMap memberMap;
 
   // Conversion to InputHandler base config
```

Comment:
InputHandler or InputBase which one should the comment refer to?

Reply:
Should have been InputBase - thanks!